pkg/store/instance.go

```diff
@@ -164,3 +168,32 @@ func ReadPIDFile(path string) (int, error) {
 	}
 	return pid, nil
 }
+
+type FormatData struct {
+	Instance
+	HostOS       string
+	HostArch     string
+	LimaHome     string
+	IdentityFile string
+}
```

Comment:
I think we have to consolidate this struct into what we have for sockets

https://github.com/lima-vm/lima/blob/7d3eca52b06ae052c5e09d5e6b8e113eab0b1ee9/pkg/limayaml/defaults.go#L197-L206

Having two similar but different structs is confusing.

Reply:
We consolidated the one from "limactl list", but didn't do anything about the socket templates